ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/StructHelper.java

```diff
@@ -163,7 +163,7 @@ public static boolean requiresNewCustomParser(final Struct struct, final Set<Str
                     for (final Struct struct : allStructs) {
                         LOG.warn("    " + struct.toString() + "\n");
                     }
-                    break;
+                    throw new ParseException("Unable to parse API Contract.", 0);
```

Comment:
I'm not sure this is an issue with parsing the contract, as much as converting it